tests/dash/configs/privatelink_config.py

```diff
@@ -151,7 +151,7 @@
 
 METER_POLICY_V4_CONFIG = {
     f"DASH_METER_POLICY_TABLE:{METER_POLICY_V4}": {
-        "ip_version": "ipv4"
+        "ip_version": "1"
```

Comment:
Can you use the protobuf enum here? You should be able to access it with `from dash_api.types_pb2 import IpVersion`

Reply:
done, thanks